klaytn-tx-latency-measurement/sendtx_klaytn.js

```diff
@@ -171,9 +211,9 @@ async function sendTx() {
         // console.log(`${data.executedAt},${data.chainId},${data.txhash},${data.startTime},${data.endTime},${data.latency},${data.txFee},${data.txFeeInUSD},${data.resourceUsedOfLatestBlock},${data.numOfTxInLatestBlock},${data.pingTime},${data.error}`)
     }
     try{
-        await uploadToS3(data)
+        await uploadChoice(data)
     } catch(err){
-        console.log('failed to s3.upload! Printing instead!', err.toString())
+        console.log('failed to gcs.upload! Printing instead!', err.toString())
```

Comment:
maybe this log can be changed like `${process.env.UPLOAD_METHOD === 'AWS'?'s3':'gcs'}.upload!`

Reply:
Just committed! Please let me know if this is what you meant